app/views/users/show.html.erb

```diff
@@ -1 +1,9 @@
-<h1>Here is a profile of a given user</h1>
\ No newline at end of file
+<%= render @user %>
+<% if @user.bio.present? %>
+	<div class="mt-5 notification">
+	  <h5 class="title is-6">Bio</h5>
+	  <p class="is-6"><%= @user.bio %></p>
+	</div>
+<% end %>
+<%= render partial: "posts/post", collection: @posts %>
+<%= link_to "See all post", user_posts_path(@user.id), class: "button is-link is-small" %>
```

Comment:
- [ ] I should see the recent post method in the user profile. Please fix it.

![image](https://user-images.githubusercontent.com/38635158/161262990-11970ab7-a60c-46ac-92c8-d4dd5b741add.png)

Reply:
Thanks You for  the review @ahmetbozaci